users_ldap_populate/tests/test_users_ldap_populate.py

```diff
@@ -63,9 +62,10 @@ def test_users_ldap_populate(self):
             'uid': ['fake'],
             'mail': ['fake@fakery.com'],
         })]):
```

Comment:
if you add a second dict here describing to a user that you add before as inactive (demo user will do), you also test the [activating inactive users codepath](https://codecov.io/gh/OCA/server-auth/compare/42b4727d9d0b9fff3dcc6690887168237818b870...e94d80c8cc3ba08efcbc85d8bcae651e1003cb99/diff#D65-71) which probably raises coveragfe enough to let this test pass

Reply:
@Tardo do you know what to do then? I confess I'm a bit lost with this stuff.

Reply:
@hbrunn Can you verify that '_get_or_create_user' can return a "false" value?
- If the user exists and is actived = The user is used
- If the user exists and is deactivated = Exception
- If the user not exists and creation is not enabled = Exception
- If the user not exists and creation is enabled = Copy/Creates a new user

Reply:
the function won't return a falsy value, but that's not the point here. Just turn on the `create_user` flag, then you actually test what this addon does.

Reply:
@hbrunn "create_user" is enabled by default: https://github.com/odoo/odoo/blob/12.0/addons/auth_ldap/models/res_company_ldap.py#L34
I don't understand you, sorry.